src/libsyntax/feature_gate.rs

```diff
@@ -688,6 +685,8 @@ declare_features! (
     (accepted, min_const_unsafe_fn, "1.33.0", Some(55607), None),
     // `#[cfg_attr(predicate, multiple, attributes, here)]`
     (accepted, cfg_attr_multi, "1.33.0", Some(54881), None),
+    // Allows `cfg(target_vendor = "...")`.
+    (accepted, cfg_target_vendor, "1.33.0", Some(29718), None),
```

Comment:
```suggestion
    (accepted, cfg_target_vendor, "1.34.0", Some(29718), None),
```
This will miss master=>beta promotion on the 15th of January.